en/docs/learn/tenant-wise-email-sender-configuration.md

```diff
@@ -20,10 +20,9 @@ We are configuring email-sender configurations by adding the following configura
  enable_start_tls= true
  enable_authentication= true
  ```
+ However, this configuration will apply to all the tenants. If you are having a requirement to configure them tenant-wise follow the instructions given below instead. 
```

Comment:
Shall we rephrase the following? WDYT

If you are having a requirement --> If you wish to configure them....